projects/storefrontapp-e2e-cypress/cypress/helpers/product-search.ts

```diff
@@ -100,18 +114,26 @@ export function sortByTopRated() {
   cy.get(firstProductNameSelector).should('not.be.empty');
 }
 
-export function checkFirstItem(title: string): void {
+export function checkFirstItem(productName: string): void {
   cy.get('cx-product-list-item .cx-product-name')
     .first()
-    .should('contain', title);
+    .should('contain', productName);
 }
-export function createDefaultQueryRoute(alias: string): void {
+
+export function createGenericQuery(alias: string): void {
```

Comment:
maybe call it createGenericProductQuery, because other e2e tests uses stubbing as well, and having it called generic is maybe false. This is only true for product-search

Reply:
If it is inside product search, it is a product query by default after the fact of where it's being imported from. I don't think we need to rename this.

Reply:
Done.